docs/tutorial/custom-window-styles.md

```diff
@@ -0,0 +1,43 @@
+# Custom window styles
+
+## Frameless windows
+
+A frameless window removes all [chrome][] applied by the OS, including window controls.
+
+To create a frameless window, set the [`BaseWindowContructorOptions`][] `frame` param in the `BrowserWindow` constructor to `false`.
+
+```js title='main.js'
+const { BrowserWindow } = require('electron')
+const win = new BrowserWindow({ frame: false })
+```
+
+## Transparent windows
+
+To create a fully transparent window, set the [`BaseWindowContructorOptions`][] `transparent` param in the `BrowserWindow` constructor to `true`.
+
+```js title='main.js'
+const { BrowserWindow } = require('electron')
+const win = new BrowserWindow({ transparent: true })
+```
```

Comment:
Would screenshots be helpful here?

Reply:
yes, for sure. added fiddles as well since the code snippets can't be run in fiddle as-is